CONTRIBUTOR_LADDER.md

```diff
@@ -20,7 +20,7 @@ Description: A Community Participant participates in the community and contribut
 * Responsibilities:
     * Must follow the [CNCF Code of Conduct]
 * How users can get involved with the community:
-    * Participating in community discussions in GitHub, Slack, and meetings
+    * Participating in community discussions in [GitHub](https://github.com/openservicemesh), [Slack](https://cloud-native.slack.com/archives/C018794NV1C), and [meetings](https://zoom.us/my/cncfosm?pwd=aXdkaGU3OWRjUllyaHZEZkh0ZjFwUT09)
```

Comment:
maybe link to https://github.com/openservicemesh/osm#community  so we don't get these out of sync?

Reply:
Yes. Make sense to me. I will make the change. Thanks！

Reply:
Link updated